lib/install/management/create.go

```diff
@@ -211,3 +213,69 @@ func (d *Dispatcher) uploadImages(files map[string]string) error {
 	}
 	return nil
 }
+
+// cleanupAfterCreationFailed cleans up the dangling resource pool for the failed VCH and any bridge network if there is any.
+// The function will not abort and early terminate upon any error during cleanup process. Error details are logged.
+func (d *Dispatcher) cleanupAfterCreationFailed(conf *config.VirtualContainerHostConfigSpec, cleanupNetwork bool) {
+	defer trace.End(trace.Begin("", d.op))
+	var err error
+
+	// clean up dangling empty resource pool
+	d.op.Info("Clean up dangling VCH resource pool after VCH creation failed...")
+
+	if err = d.cleanupEmptyPool(conf); err != nil {
```

Comment:
I'd suggest splitting this into two lines. This syntax is useful to initialize a variable used in a conditional, but can be less clear in cases like this where the effects of the code "hidden in" the conditional are the primary point of the method.